### ISSUE TITLE
How to add my JS file?

### ISSUE THREAD

**Issue by Ikarushka** [thedevdojo/voyager#2846]
- Laravel Version: 5.5.#
- Voyager Version: 1.0.#
- PHP Version: 7
- Database Driver & Version:

### Description:
I need to add my js file to my view. I read docs https://voyager.readme.io/docs/additional-css-and-js-theming an do that:
I copied my file **mode-php.js** to folder  **vendor/tcg/voyager/resources/assets/js**
I changed file **publishable/config/voyager.php** :

```
 'additional_js' => [
        'js/mode-php.js',
],
```

I maked link to my master.blade.php:
`<script type="text/javascript" src="{{ voyager_asset('js/mode-php.js') }}"></script> `

But browser get 404 for **/vendor/tcg/voyager/assets/js/mode-php.js**
Tell me please where I'm wrong?

**Comment by emptynick** [comment]
First: dont change anything inside `vendor`.
Open your `config/voyager.php` file, and add your path `js/mode-php.js` to `additional_js`.
Then place your file as `public/js/mode-php.js`
Thats it. You dont need to edit any views for it

**Comment by Ikarushka** [comment]
Thanks! 
I did so, but don't work.  I checked  it in **master.blade.php** by 
`<?php print_r (config('voyager.additional_js')); ?>`
and I get empty array. But I'm not sure what I'm checking properly. I can't understand why my correct **config/voyager.php**    
```
 'additional_js' => [
        'js/mode-php.js',
    ],
```
don't work in view. I just start whith Voyager and Laravel. How to get array with Voyager config for checked it?
